lib/improver/wxcode/weather_symbols.py

```diff
@@ -260,18 +278,28 @@ def construct_extract_constraint(diagnostics, thresholds):
             iris.Constraint or list of iris.Constraints:
```

Comment:
Does this not return a string or list of strings now?

Reply:
:white_check_mark: Yep, I've changed this in a few places now.